_maps/yogstation/RandomRuins/StationRuins/BoxStation/engine_singulo.dmm

```diff
@@ -0,0 +1,1707 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```

Comment:
could you be a doll and put external airlocks at the bottom corners of the singulo pen, like on prebase?

Reply:
done